Plugins/GeoModel/include/Acts/Plugins/GeoModel/GeoModelConverters.hpp

```diff
@@ -68,4 +69,24 @@ using GeoUnionDoubleTrdConverter =
     detail::GenericGeoShapeConverter<GeoShapeUnion,
                                      detail::GeoUnionDoubleTrdConverter>;
 
+/// @brief The map that maps the converters with the shapes
+
+std::shared_ptr<const IGeoShapeConverter> GeoShapesConverters(int geoShapeId) {
+  using ConverterLookUpMap =
+      std::map<int, std::shared_ptr<const IGeoShapeConverter>>;
```

Comment:
`std::unordered_map` or `boost::flat_map` would be better suited the use case here I think.

In the future, I think there might be the possibilities that you have multiple matching converters per ShapeType (e.g., multiple union converter), so than we would need a `multi_map` and return a `vector`... but I think we can worry about this in the future when it happens.

Reply:
changed to unorder_map!